zpay32/invoice_test.go

```diff
@@ -17,9 +17,7 @@ import (
 	"github.com/btcsuite/btcd/btcutil"
 	"github.com/btcsuite/btcd/chaincfg"
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
-	"github.com/btcsuite/btcd/wire"
```

Comment:
was this ever needed? Seems unrelated to litecoin removal but I could be missing something.

Reply:
you mean the `"github.com/btcsuite/btcd/wire"` import? it was used by the litecoin tests. see below. Golang doesnt allow you to have an import that you dont use unless you explicitly prefix it with `_`